compiler/src/iree/compiler/Codegen/Common/MaterializeTuningSpecsPass.cpp

```diff
@@ -143,15 +143,12 @@ getDefaultTuningSpec(ModuleOp module,
       dialect.getOrParseTransformLibraryModule(defaultTuningSpecName,
                                                *defaultTuningSpecSource);
 
-  if (failed(defaultTransformLibrary)) {
-    return module->emitError()
-           << "Failed to load  default tuning spec" << defaultTuningSpecName;
-  }
-
 #ifndef NDEBUG
-  if (failed(mlir::verify(*defaultTransformLibrary)))
+  if (succeeded(defaultTransformLibrary) &&
+      failed(mlir::verify(*defaultTransformLibrary)))
     return (*defaultTransformLibrary).emitError()
-           << "Verification failed for default tuning spec";
+           << "Default tuning spec " << defaultTuningSpecName
+           << " failed to verify";
```

Comment:
Have you confirmed that this error gets emitted when the default spec is invalid?

Reply:
@bangtianliu can you please reply instead of marking this as resolved? I don't know what the outcome of this is.

Reply:
Yes, I did check it and fixed the crashes.